Products/ZenUI3/browser/resources/js/zenoss/AddToZenPackWindow.js

```diff
@@ -68,15 +68,15 @@ Ext.define("Zenoss.AddToZenPackWindow", {
                     forceSelection: true,
                     triggerAction: 'all',
                     selectOnFocus: true,
-                    id: 'zpcombobox'
+                    itemId: 'zpcombobox'
                 }],
                 buttons: [{
                     text: _t('Submit'),
                     xtype: 'DialogButton',
                     disabled: true,
-                    handler: function () {
-                        var form = Ext.getCmp('addzenpackform'),
-                            chosenzenpack = form.getForm().findField('zpname').getValue(), i, targets = [], callback;
+                    handler: function (t) {
```

Comment:
go up to button owner (form) and down (get it child "zpcombobox") instead of searching in all form fields.